tasmota/tasmota_xsns_sensor/xsns_114_amsx915.ino

```diff
@@ -0,0 +1,251 @@
+/*
+  xsns_114_ams.ino - AMS5915, AMS6915 pressure and temperature sensor support for Tasmota
+
+  Copyright (C) 2023 Bastian Urschel
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifdef USE_I2C
+#ifdef USE_AMSX915
+/*********************************************************************************************\
+ * AMS5915, AMS6915 - Pressure and Temperature
+ *
+ * Source: Bastian Urschel
+ *
+ * I2C Address: 0x28
+\*********************************************************************************************/
+
+#define XSNS_114            114
+#define XI2C_86              86  // See I2CDEVICES.md
+
+#ifndef AMSX915_ADDR
+  #define AMSX915_ADDR         0x28
+#endif
+
+#define PMIN_DEFAULT      0
+#define PMAX_DEFAULT      0
+
+#ifndef USE_UFILESYS
+#warning "AMSx915 pressure settings cannot be saved persistent due to missing filesystem"
+#endif
+
+/********************************************************************************************/
+
+struct amsx915_data {
+  float pressure;
+  float temperature;
+  bool present;
+} amsx915data;
+
+const uint16_t AMSX915_VERSION = 0x0100;       // Latest sensor version (See settings deltas below)
+
+// Global structure containing driver saved variables
+struct {
+  uint32_t  crc32;    // To detect file changes
+  uint16_t  version;  // To detect driver function changes
+  int16_t   pmin;
+  int16_t   pmax;
+} amsx915Settings;
+
+void AmsDetect(void) {
+  // sensor datasheets does not provide any information/commands on sensor
+  // specific register so it is not possible to properly detect sensor type
+  if(Wire.available() != 4) {
+    // Error
+    amsx915data.present = false;
+    AddLog(LOG_LEVEL_INFO, PSTR("CFG: AMS not detected!"));
```

Comment:
You shouldn't log at `INFO` level any undetected device, otherwise the logs would be crowded by all the potential devices you may connect. Please remove, or at least demote to `LOG_LEVEL_DEBUG`